### ISSUE TITLE
[Segmented] Narrator doesn't announce invoked action

### ISSUE THREAD

**Issue by niels9001** [CommunityToolkit/Windows#193]
**Pre-Requisite:**
Turn on narrator with shortcut key "win+ctrl+Enter" key.

**Repro-Steps:**
Launch the WinUI app. 
Navigate to search box using tab key.
Type “Colors” and hit enter.
Navigate to Colors card and activate it.
Now navigate to any banner items i.e., Text, Fill, Stroke, Background invoke it and observe.
Actual Result:

When user invoke any of the banner items screen reader fails to announce the success information of the activate item. Narrator remains silent on invoking any of the button (Text, Fill, Stroke, Background).

 
**Expected Result:**
Screen reader should not remain silent when user invoke any of the button (Text, Fill, Stroke, Background). Narrator announce related to the selected button.

 
**User Impact:**
User with visual impairments who rely on screen reader will face difficulties and will not be able to know whether the invoked button is selected or not if screen reader fails to announce success information.
